bentoml/cli/bento_management.py

```diff
@@ -259,7 +259,7 @@ def retrieve(bento, target_dir):
 
         if get_bento_result.bento.uri.s3_presigned_url:
             bento_service_bundle_path = get_bento_result.bento.uri.s3_presigned_url
-        if get_bento_result.bento.uri.gcs_presigned_url:
+        elif get_bento_result.bento.uri.gcs_presigned_url:
```

Comment:
add a test case?  and why does it has both s3_presigned_url and gcs_presigned_url?  It should not be possible, right?